src/scripts/main.js

```diff
@@ -5,4 +5,42 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const mainTab = document.querySelector('.dashboard');
+
+for (let i = 0; i < people.length; i++) {
+  const tab = document.createElement('tr');
```

Comment:
It would make more sense for this to be named `tableRow` or `tr` rather than `tab`